### PR TITLE
Added support for custom cpu share

### PR DESCRIPTION
**Pull request by santhosh-sloka**
To Support CPU share in case " https://github.ibm.com/zvc/planning/issues/18693 " 

--signed off : ssanthosh@in.ibm.com

### CODE REVIEW

**Comment by santhosh-sloka** [comment]
Dear @SeanHQF @jichenjc - This is the dependency PR for " https://github.ibm.com/zvc/nova-icic/pull/102 ", will do final test and update here for review and merge, thanks.

**Comment by SeanHQF** [comment]
@bjhuangr pls review

**Comment by jichenjc** [comment]
@santhosh-sloka the code you made is inconsistent to the code
code is for `comments` and you said here it's for cpu share

please confirm

**Comment by santhosh-sloka** [comment]
> @santhosh-sloka the code you made is inconsistent to the code code is for `comments` and you said here it's for cpu share
> 
> please confirm

hi @jichenjc - As this is part of the CPU share feature, I have added that comment, moreover we are changing 2 files, in one we are adding the comments, in the validation parameter file, we are adding support to include customer cpu share ..

File: zvmsdk/sdkwsgi/handlers/guest.py [hunk 1]
```diff
@@ -84,6 +84,8 @@ def create(self, body):
             kwargs_list['rdomain'] = guest['rdomain']
         if 'pcif' in guest_keys:
             kwargs_list['pcif'] = guest['pcif']
+        if 'comments' in guest_keys:
+            kwargs_list['comment_list'] = guest['comments']
```
Comment (jichenjc) [inline zvmsdk/sdkwsgi/handlers/guest.py:88]: @santhosh-sloka I am talking about this ,I don't know why this is related to CPU share at all

Comment (santhosh-sloka) [reply zvmsdk/sdkwsgi/handlers/guest.py:88]: @jichenjc - This is one of the requirement in custom CPU share enablement case where we enable customer to add some comments for their reference in the extra_specs

**Comment by santhosh-sloka** [comment]
@jichenjc @SeanHQF - We are testing the existing scenario , will confirm by this week or the merger of PR : https://github.com/openmainframeproject/feilong/pull/834, https://github.com/openmainframeproject/feilong/pull/833 and https://github.ibm.com/zvc/nova-icic/pull/102

**Review by SeanHQF — APPROVED** [review]

**Comment by santhosh-sloka** [comment]
Update :

Have added validation of share ABS and all its combination, REL and all its combination, ABS value to validate decimal etc . 

This is good to merge, we have finalised the code for " https://github.ibm.com/zvc/nova-icic/pull/102 " correcting some UT failures, once fixed both can be merged at one go, thanks .. 

@SeanHQF , @jichenjc

**Comment by santhosh-sloka** [comment]
@bjhuangr - FYI ..

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**